### PR TITLE
TMA-536: Release brick: transfer Dataset and Fact Tags in ImportObjectCollection

### PR DESCRIPTION
**Pull request by vhtien**
## Todos
- [ ] Tests
- [ ] Documentation
- [ ] Changelog
- [ ] Build passing

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/829/ : FAILURE in 7s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/785/ : SUCCESS in 44m 31s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/557/ : FAILURE in 25s

**Comment by vhtien** [comment]
ok to test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/835/ : SUCCESS in 14s
- https://ci.intgdc.com/job/gooddata-ruby-tests-zuul/791/ : SUCCESS in 44m 50s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/563/ : UNSTABLE in 14m 09s

**Comment by vhtien** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/33/ : SUCCESS in 5s
- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/860/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/7/ : SUCCESS in 2m 44s

**Comment by vhtien** [comment]
extended test

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/36/ : SUCCESS in 5s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/809/ : SUCCESS in 41m 37s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/582/ : UNSTABLE in 1m 36s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/45/ : SUCCESS in 5s
- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/883/ : SUCCESS in 16s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/30/ : SUCCESS in 2m 43s

File: lib/gooddata/lcm/actions/collect_ldm_objects.rb [hunk 1]
```diff
@@ -28,7 +28,7 @@ def call(params)
           synchronize = params.synchronize.pmap do |info|
             from = info.from
             from_project = development_client.projects(from) || fail("Invalid 'from' project specified - '#{from}'")
-            objects = (from_project.attributes.to_a + from_project.labels.to_a).map(&:uri)
+            objects = (from_project.attributes.to_a + from_project.labels.to_a + from_project.datasets.to_a + from_project.facts.to_a).map(&:uri)
```
Comment (panjan) [inline lib/gooddata/lcm/actions/collect_ldm_objects.rb]: We should check that datasets and facts get transferred in the AppStore integration test (/spec/integration/shared_examples_for_synchronization_bricks.rb).

**Comment by vhtien** [comment]
That spec is in gerrit, not in github. I think we will need another PR for it.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-opensource-repository-check/69/ : SUCCESS in 5s
- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/907/ : SUCCESS in 17s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/54/ : SUCCESS in 14s

**Comment by vhtien** [comment]
Integration tests: https://ms-gerrit.intgdc.com/#/c/627/

**Comment by panjan** [comment]
ok to merge

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://ci.intgdc.com/job/gooddata-ruby-merge-compliance-check/29/ : SUCCESS in 14s
- https://ci.intgdc.com/job/gooddata-ruby-rubocop-zuul/908/ : SUCCESS in 21s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/55/ : SUCCESS in 19s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/837/ : SUCCESS in 44m 32s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/610/ : SUCCESS in 22m 56s

**Pull request merged**